.travis.yml

```diff
@@ -1,9 +1,6 @@
 language: node_js
 sudo: false
 node_js:
-- '0.10'
-- '0.12'
-- '2.0.0'
 - '4'
```

Comment:
Since we aren't going to make this PR as a breaking change, it would make sense to keep these versions here.